recipe/meta.yaml

```diff
@@ -22,15 +25,16 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - cmake >=3.8
-    - ninja
+    - cmake >=3.18
+    # we currently don't have ninja 1.11 but it's building and passing tests
+    - ninja >=1.10.2
```

Comment:
This is for all platforms - do you want to use the ninja generator on windows, or put a unix selector after this?

Reply:
There were no selectors in the previous version of the recipe, so I am unaware if it's required or not. Do I need to add selectors for this?

Reply:
> Should I add selectors for this?

I think it would be better to add a `# [unix]` selector for this requirement if it's currently working. My point is, you can see in the build command that on windows, the CMake generator, which generates the build system, is of the type "visual studio", not "ninja". So it would seem that ninja isn't needed on windows.

An alternative could be to use ninja as the build system, which could be better as ninja's fast. But it might also not work and generate extra work for you.

Reply:
added selector to ninja